docs/ref/contrib/gis/db-api.txt

```diff
@@ -310,41 +310,41 @@ divided into the three categories described in the :ref:`raster lookup details
-:lookup:`within`                   X          X         X            X          B
-:lookup:`left`                     X                                            C
-:lookup:`right`                    X                                            C
-:lookup:`overlaps_left`            X                                            B
-:lookup:`overlaps_right`           X                                            B
-:lookup:`overlaps_above`           X                                            C
-:lookup:`overlaps_below`           X                                            C
-:lookup:`strictly_above`           X                                            C
-:lookup:`strictly_below`           X                                            C
-=================================  =========  ========  ============ ========== ========
+=================================  =========  ======== ========= ============ ========== ========
+Lookup Type                        PostGIS    Oracle   MariaDB   MySQL [#]_   SpatiaLite PGRaster
+=================================  =========  ======== ========= ============ ========== ========
+:lookup:`bbcontains`               X                   X         X            X          N
+:lookup:`bboverlaps`               X                   X         X            X          N
+:lookup:`contained`                X                   X         X            X          N
+:lookup:`contains <gis-contains>`  X          X        X         X            X          B
+:lookup:`contains_properly`        X                                                     B
+:lookup:`coveredby`                X          X                               X          B
+:lookup:`covers`                   X          X                               X          B
+:lookup:`crosses`                  X                   X         X            X          C
+:lookup:`disjoint`                 X          X        X         X            X          B
+:lookup:`distance_gt`              X          X        X         X            X          N
+:lookup:`distance_gte`             X          X        X         X            X          N
+:lookup:`distance_lt`              X          X        X         X            X          N
+:lookup:`distance_lte`             X          X        X         X            X          N
+:lookup:`dwithin`                  X          X                               X          B
+:lookup:`equals`                   X          X        X         X            X          C
+:lookup:`exact`                    X          X        X         X            X          B
+:lookup:`intersects`               X          X        X         X            X          B
+:lookup:`isvalid`                  X          X                  X (≥ 5.7.5)  X (LWGEOM)
+:lookup:`overlaps`                 X          X        X         X            X          B
+:lookup:`relate`                   X          X                               X          C
```

Comment:
MariaDB supports `ST_Relate` (see [documentation](https://mariadb.com/kb/en/library/st_relate/)) but it is not supported in Django. I'm going to fix this and update documentation in Django 3.1 because this PR is targeted for a backport to Django 3.0.